packages/terra-tag/tests/nightwatch/DefaultTag.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
```

Comment:
Can we either 
1. (**Advised**) Change the name of this file so the user doesn't expect only the `DefaultTag` defined below to be within
2. Move all the examples below to separate files.

Reply:
We should move each example into their own file. When we migrate to use the terra-dev-site, these will need to be their own.

Reply:
Updated it [here](https://github.com/cerner/terra-core/pull/1306/commits/a6617b0978ac0a4b8bfd27316968c1543d5be2df).